docs/dev-docs-for-loader-customization.rst

```diff
@@ -0,0 +1,69 @@
+.. _loader_customization_guide:
+
+===============================
+Data Loader Customization Guide
+===============================
+
+The `glue` data visualization tool is highly customizable for different data types, such as for FITS files
+that are often used in solar physics research. There are two ways to customize `glue` using the `glue-solar` plugin:
+
+1. The first option is to develop a custom loader for the solar data taken from practically any (ground-based or
+satellite-borne) instrument.
+
+2. The second option is to develop a custom viewer to enable a certain very specific way to view your data, such as
+an alternative of the existing 2D image viewer that presents your data in a manner you like; e.g. color-coding the
```

Comment:
Do we have a better example?

Reply:
Not for solar physics not at the moment...

Reply:
That is fine.